tableau_exporter/tableau_status_exporter.py

```diff
@@ -13,15 +13,28 @@
 logger = logging.getLogger(__name__)
 
 class TokenManager(object):
-    def __init__(self, user, password, site, host, api_version):
+    def __init__(self, host, api_version, site, user, password, token_name, token_secret):
```

Comment:
Please keep the original argument ordering for backwards compatibility, and token_name, token_secret to None (maybe also user and password, but that's not needed for backwards compatibility)